benchmark/realnest/08_count_map_keys.benchmark

```diff
@@ -0,0 +1,19 @@
+# name: benchmark/realnest/08_count_map_keys.benchmark
+# description: Count map keys and aggregate them
+# group: [realnest]
+
+name count_map_keys
+group real_nest
+
+require json
+
+require httpfs
+
+load benchmark/realnest/load.sql
+
+run
+SELECT keys, count(*) mentions FROM
+    (SELECT UNNEST(map_keys(tags)) AS keys FROM daylight)
```

Comment:
also the name of the table here needs to match the table name in `load.sql`. So either `daylight` here needs to change to `open_street_map` or `open_street_map` in the load.sql needs to change to `daylight`